modules/tinymce/src/plugins/lists/test/ts/browser/BackspaceDeleteFromBlockIntoLiTest.ts

```diff
@@ -126,4 +126,12 @@ describe('Browser Test: .RemoveTrailingBlockquoteTest', () => {
     TinyContentActions.keystroke(editor, Keys.backspace());
     TinyAssertions.assertContent(editor, '<ul><li>a</li></ul>');
   });
+
+  it('TINY-6888: delete a `li` with a `br` and a `br` with `data-mce-bogus', () => {
```

Comment:
Could you add some tests with non bogus BRs as well so that we don't break `<li>foo<br><br>` or `<li>foo<br><br><br>` cases? Or any other cases you can think of.

Reply:
added